aclogview/CM_Movement.cs

```diff
@@ -664,6 +669,40 @@ public class Jump_NonAutonomous : Message {
         }
     }
 
+    public class LifestoneMaterialize : Message
+    {
+        public uint ObjectId;
+        public int unknown1;
+        public Position Position;
+        public int unknown2;
+        public int unknown3;
+
+        public static LifestoneMaterialize read(BinaryReader binaryReader)
+        {
+            LifestoneMaterialize newObj = new LifestoneMaterialize();
+            newObj.ObjectId = binaryReader.ReadUInt32();
+            newObj.unknown1 = binaryReader.ReadInt32();
+            newObj.Position = Position.read(binaryReader);
+            newObj.unknown2 = binaryReader.ReadInt32();
+            newObj.unknown3 = binaryReader.ReadInt32();
```

Comment:
Thanks for the changes. Was intending for two _more_ ints after position though - id, unk1, pos, unk2, unk3, unk4, unk5, unk6, unk7. Naturally we have no idea if those are actually ints, but at least the whole fragment gets read that way and you won't get that warning.